src/main/scala/com/monsanto/arch/cloudformation/model/resource/EC2.scala

```diff
@@ -78,6 +78,20 @@ object `AWS::EC2::KeyPair::KeyName` extends DefaultJsonProtocol {
   implicit val format: JsonFormat[`AWS::EC2::KeyPair::KeyName`] = jsonFormat2(`AWS::EC2::KeyPair::KeyName`.apply)
 }
 
+case class `AWS::EC2::CustomerGateway`(
+  name: String,
+  BgpAsn: Int,
+  IpAddress: String,
```

Comment:
Can you make this more typesafe by using the new `IPAddress` type ( see https://github.com/MonsantoCo/cloudformation-template-generator/blob/master/src/main/scala/com/monsanto/arch/cloudformation/model/resource/EC2.scala#L235-L248).